src/ClusterBootstrap/deploy.py

```diff
@@ -3408,6 +3408,12 @@ def run_command(args, command, nargs, parser):
         if nargs[0] == "config":
             update_config_nodes()
 
+    elif command == "updatemysqlvc":
+        from MySQLDataHandler import update_db
+        with open("deploy/RestfulAPI/config.yaml") as f:
```

Comment:
Why loading `config.yaml` again? All configurations should already be in `config.yaml`?

Reply:
no, it's different yaml file. this MysqlDatahandler.py reads the config in RestfulAPI container, as in the code. deploy.py instead only reads ClusterBootstrap/config.yaml and cluster.yaml, if we don't read the one in the code, there would be key error